docs/developer-docs/production/social-sharing.md

```diff
@@ -0,0 +1,80 @@
+# Sharing links to dapps
+
+## Overview
+
+Since service workers are not supported in all settings (especially on mobile),
+it is important to use the `raw` links (e.g., [oa7fk-maaaa-aaaam-abgka-cai.raw.icp0.io](https://oa7fk-maaaa-aaaam-abgka-cai.raw.icp0.io))
+when sharing links on different platforms (e.g., LinkedIn or Twitter/X). This guide
+provides the necessary background and explains how to configure your asset
+canister correctly, such that you can share links to your dapp without hiccups.
+
+## Context
+
+To directly access dapps hosted on the Internet Computer via a browser,
+an HTTP gateway is required. This gateway translates browser HTTP requests into API canister calls,
+is required. Currently, there are two instances of this HTTP gateway, providing
+users with two ways to access dapps on the IC through their browsers:
+
+1. `non-raw`: upon the initial dapp access, a service worker is loaded
+into the user's browser, which performs the translation locally. `non-raw` is available
+through the `<canister_id>.icp0.io` domains (e.g., [oa7fk-maaaa-aaaam-abgka-cai.icp0.io](https://oa7fk-maaaa-aaaam-abgka-cai.raw.icp0.io))
+or a custom domain (e.g., [internetcomputer.org](https://internetcomputer.org)).
+
+2. `raw`: the translation happens remotely on the boundary nodes and is transparent
+to the user. `raw` access is available through the `<canister_id>.raw.icp0.io` domains
+(e.g., [oa7fk-maaaa-aaaam-abgka-cai.raw.icp0.io](https://oa7fk-maaaa-aaaam-abgka-cai.raw.icp0.io)).
+
+Service workers are not universally supported, especially on mobile devices.
+In-App Web-Views, Firefox Privacy Mode, and Google translate, for example, do not
+function with service workers and thus do not support `non-raw` dapp access.
+
+This is relevant when sharing dapp links on other platforms like
+LinkedIn or Twitter. When users try to open a `non-raw` link, for example, from
+within the LinkedIn dapp, it is opened in an In-App Web-View, which does not
+support service workers and therefore simply displays an error message.
+
+:::caution
+For custom domains there is no `raw` equivalent. If you want to share a link,
+you should use the corresponding `<canister_id>.raw.icp0.io` domain.
+:::
+
+
+## Solution
+
+To enable universal dapp access through a shared link, it is
+essential to share the `raw` version (i.e., `<canister_id>.raw.icp0.io`) to bypass
+the service worker.
+
+However, two key details must be noted:
+
+1. Starting from `dfx 0.13.1`, asset canisters redirect requests for `raw` to
```

Comment:
maybe worth noting here as an alternative to use the new dfx for new canisters since they've already changed it there https://github.com/dfinity/sdk/releases/tag/0.14.4-beta.0